machine.js

```diff
@@ -301,11 +301,15 @@ function Zeal8bitComputer() {
     }
 
     function restart() {
+        running = false;
```

Comment:
Good catch! This is required indeed.
It is also possible to clear the  `interval`, which is responsible for running the machine loop.
Maybe a more sturdy solution would be to call `stop()` which does all of that.

To prevent it from showing the CPU registers maybe we can add a boolean parameter?

Reply:
Ok，I‘ll fix it.

Reply:
In fact, the stop function only has 4 lines😂 I think we can just copy the part used to empty the interval to the restart function.

Reply:
You are right, it is better like this

Reply:
You are right, it is better like this